src/pages/workspace/expensifyCard/WorkspaceExpensifyCardPage.tsx

```diff
@@ -0,0 +1,172 @@
+        name: 'Test 2',
+        limit: 2000,
+        lastFourPAN: '1234',
+    },
+    test3: {
+        // @ts-expect-error TODO: change cardholder to accountID
+        cardholder: {accountID: 3, lastName: 'Brown', firstName: 'Kevin', displayName: 'Kevin Brown', avatar: ''},
+        name: 'Test 3',
+        limit: 3000,
+        lastFourPAN: '1234',
+    },
+};
+
+function WorkspaceExpensifyCardPage({route}: WorkspaceExpensifyCardPageProps) {
+    const {shouldUseNarrowLayout} = useResponsiveLayout();
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const policyID = route.params.policyID;
+    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`);
+
+    const policyCurrency = useMemo(() => policy?.outputCurrency ?? CONST.CURRENCY.USD, [policy]);
+
+    // TODO: uncomment the code line below to use cardsList data from Onyx when it's supported
+    // const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.EXPENSIFY_CARDS_LIST}${policyID}_Expensify Card`);
+    const cardsList = mockedCards;
+
+    const fetchExpensifyCards = useCallback(() => {
+        // TODO: uncomment when OpenPolicyExpensifyCardsPage API call is supported
+        // Policy.openPolicyExpensifyCardsPage(policyID);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [policyID]);
+
+    useFocusEffect(fetchExpensifyCards);
+
+    const sortedCards = useMemo(
+        () =>
+            Object.values(cardsList ?? {}).sort((a, b) => {
+                // TODO: change cardholder to accountID and get personal details with it
+                const aName = PersonalDetailsUtils.getDisplayNameOrDefault(a.cardholder);
+                const bName = PersonalDetailsUtils.getDisplayNameOrDefault(b.cardholder);
+                return localeCompare(aName, bName);
+            }),
+        [cardsList],
+    );
+
+    const getHeaderButtons = () => (
+        <View style={[styles.w100, styles.flexRow, styles.gap2, shouldUseNarrowLayout && styles.mb3]}>
+            <Button
+                medium
+                success
+                onPress={() => {}} // TODO: add navigation action when card issue flow is implemented (https://github.com/Expensify/App/issues/44309)
+                icon={Expensicons.Plus}
+                text={translate('workspace.expensifyCard.issueCard')}
+                style={shouldUseNarrowLayout && styles.flex1}
+            />
+            <Button
+                medium
+                onPress={() => {}} // TODO: add navigation action when settings screen is implemented (https://github.com/Expensify/App/issues/44311)
+                icon={Expensicons.Gear}
+                text={translate('common.settings')}
+                style={shouldUseNarrowLayout && styles.flex1}
+            />
+        </View>
+    );
+
+    const renderItem = ({item, index}: ListRenderItemInfo<Card>) => (
+        <OfflineWithFeedback
+            key={`${item.name}_${index}`}
```

Comment:
The card name will be in card.nameValuePairs.cardTitle

https://github.com/Expensify/App/blob/3622deb409a4cf91a47cc9196e2414ffe75f0f16/src/types/onyx/Card.ts#L39-L40

You can update that comment in Card.ts to state `User-defined nickname for the card` since card titles is now not only for virtual cards